### ISSUE TITLE
problems bootstrapping 3.5.0

### ISSUE THREAD

**Issue by bsi-lz** [NCSC-NL/taranis3#20]
following your bootstrapping documentation for a fresh 3.5.0 - i get following error:

[root@taranis taranis]# ./taranis-bootstrap taranis-3.5.0.tar.gz
*** bootstrap installation of taranis-3.5.0.tar.gz
...
*** running /home/taranis/sources/taranis-3.5.0/install/120.profile
*   creating login version switch in /home/taranis/lib/bash_profile
ERROR: cannot read from /home/taranis/etc/setup-3.5.0.tar.gz.json for JSON: file or folder not found

I ran into similar problems when trying to install via git. The JSON-file creation/naming function seems to be broken...

Can you provide an example bash-profile for several installed version under same username? Then i could ignore all install-routines and just copy your code-changes into my codebase and edit the bash-profile manually.

**Comment by bsi-lz** [comment]
i didnt't change a thing, deleted everything, tried bootstrap again and now the message says:
[root@localhost home]# ./taranis-bootstrap taranis-3.5.0.tar.gz 
*** bootstrap installation of taranis-3.5.0.tar.gz
...
*** running /home/taranis/sources/taranis-3.5.0/install/120.profile
*   creating login version switch in /home/taranis/lib/bash_profile
ERROR: cannot read from /home/taranis/etc/setup-bootstrap.json for JSON: file or folder not found

**Comment by bsi-lz** [comment]
hmkay...started all over and managed to bootstrap 3.5.0. Next i will try to git update to 3.5.1...

**Comment by bsi-lz** [comment]
when trying to update to 3.5.1 via git i run again in dead-ends. This is what i have done:

tara$ taranis git init
> no ssh rights

// ok so lets try "https"
tara$  git clone https://github.com/NCSC-NL/taranis3.git ~taranis/sources/taranis-git
> all good

ok. now i am following [installing-via-git](https://github.com/NCSC-NL/taranis3/wiki/Install-Bootstrapping#installing-via-git) starting from step 2 as mentioned in [init-git-before-341-rc3-broken](https://github.com/NCSC-NL/taranis3/wiki/Install-Bootstrapping#init-git-before-341-rc3-broken):
tara$  taranis install
```
*   start installing version 3.5.0
...
*** running /home/taranis/sources/taranis-3.5.0/install/120.profile
*   creating login version switch (with git) in /home/taranis/lib/bash_profile
```

> ERROR: cannot read from /home/taranis/etc/setup-git.json for JSON: file or folder not found
> Installation stopped

**so again the step "adds GIT-setup to login" does not work for me**

**Comment by markov2** [comment]
* bsi-lz (notifications@github.com) [190214 16:20]:
> when trying to update to 3.5.1 via git i run again in dead-ends. This
> is what i have done:
> 
> tara$ taranis git init
> > no ssh rights

Why are you not resolving the ssh rights issue?

> *   start installing version 3.5.0

That's not the right one.
Go to: ~taranis/sources/taranis-git
Type:   bin/taranis install           # should work withoun bin/ in 3.6
>
>> ERROR: cannot read from /home/taranis/etc/setup-git.json for JSON: file or folder not found

Still running the old install, so again hit the same bug.

I agree that it is a bit painful.  My collegues managed to get it
to work with the same instructions
   taranis git init
   cd sources/taranis-git
   bin/taranis install

-- 
               MarkOv

------------------------------------------------------------------------
drs Mark A.C.J. Overmeer                                MARKOV Solutions
       Mark@Overmeer.net                          solutions@overmeer.net
http://Mark.Overmeer.net                   http://solutions.overmeer.net

**Comment by bsi-lz** [comment]
> bin/taranis install

Thanks alot, why is that missing in the docs? :D 
Sorry, i am a linux noob...

**Comment by markov2** [comment]
* bsi-lz (notifications@github.com) [190214 17:07]:
> > bin/taranis install
> 
> Thanks alot, why is that missing in the docs? :D 
> Sorry, i am a linux noob...

Because it did not behave as expected.  In the newer release, you do
not need to use "bin/", just be in the right directory.

Some of these paths configurations are quite tricky.  Each time
something else breaks.  Getting there (I hope)
-- 
Regards,
               MarkOv

------------------------------------------------------------------------
       Mark Overmeer MSc                                MARKOV Solutions
       Mark@Overmeer.net                          solutions@overmeer.net
http://Mark.Overmeer.net                   http://solutions.overmeer.net

**Comment by markov2** [comment]
fixed with the release of 3.6
